.travis.yml

```diff
@@ -0,0 +1,37 @@
+---
+dist: trusty
+os: linux
+language: ruby
+cache: bundler
+rvm:
+  - 2.4.5
+  - 2.5.8
+  - 2.6.5
+  - 2.7.1
+  - ruby-head
+gemfile:
+  - gemfiles/rails_4.gemfile
+  - gemfiles/rails_5.gemfile
+  - gemfiles/rails_6.gemfile
+jobs:
+  fast_finish: false
+  exclude:
+    - gemfile: gemfiles/rails_5.gemfile
+      rvm: 2.4.5
+    - gemfile: gemfiles/rails_6.gemfile
+      rvm: 2.4.5
+  allow_failures:
+    - rvm: ruby-head
+install:
+  - gem env
+  - gem list bundler
+  - rm -f /home/travis/.rvm/rubies/ruby-2.*/lib/ruby/gems/2.*/specifications/default/bundler-2.*.gemspec
```

Comment:
Uggh! Ruby 2.7 now ships with bundler 2.0, set as a "default". We aren't ready to move to 2.0 just yet. So this hack uninstalls it so we can install 1.17.3 below and have it take precedence.